alphalens/tears.py

```diff
@@ -393,6 +395,22 @@ def create_returns_tear_sheet(factor_data,
                                            ylim_percentiles=(5, 95),
                                            ax=ax_quantile_returns_bar_by_group)
 
+    ff_factors = ep.utils.load_portfolio_risk_factors() \
+        .drop(['RF'], axis='columns')
+    hierarchy = OrderedDict([
+        # not typos! These are the Fama-French factors names from Dartmouth
+        ('Market', ['Mkt-RF']),
+        ('Style', ['SMB', 'HML', 'Mom'])
+    ])
+
+    factor_returns.index = factor_returns.index.tz_localize('UTC')
```

Comment:
Alphalens code can deal with both tz_localized and non-tz_localized input. This statement force factor_returns to be tz naive because it breaks when factor_returns is tz_localized already.